curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,434 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a common method for hiding content in HTML?
```

Comment:
I think this question needs to clarify what type of hiding is being done. Is this "visually hiding" content from sighted people while still making the content accessible to assistive technology? If so, then only `opacity: 0` accomplishes that. `display: none` doesn't just "hide" content, it completely removes it from the DOM. While `visibility: hidden` does visually hide the content and leaves it in the DOM but removes it from the accessibility tree, which removes it for people using assistive technology, such as screen readers.